src/components/ListItem.jsx

```diff
@@ -1,9 +1,49 @@
 import './ListItem.css';
+import { useEffect, useCallback } from 'react';
+import { updateItem } from '../api/firebase';
+
+const millisecondsIn24hrs = 86400000;
+const currentDate = new Date();
+const currentTime = currentDate.getTime();
```

Comment:
these two allong with the `calcDate` should be inside the useEffect function on there. If we have them there the will be calculated only once when the file loads, not on the first render

Reply:
Done